# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/PUtilityProcess.ipdl
# Commit: af562cec3d70
# Full Hash: af562cec3d706ee3ed605ffeade4d3ba982d2545
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-10-27 04:08:13
# Regressor Bug: 1837079
# File Overlap Count: 2
# Description:
#   Bug 1837079 - [5/10] Create WinFileDialog IPDL protocol and implementation  r=gstoll,handyman,ipc-reviewers,nika,win-reviewers,mhowell
#   
#   Create and implement a new top-level IPC protocol, `PWinFileDialog`,
#   using the primitives from bug 1833450 and the new sandboxing type from
#   the previous commit.
# ==============================================================================

diff -r b5d52e5f75f2 -r af562cec3d70 ipc/glue/PUtilityProcess.ipdl
--- a/ipc/glue/PUtilityProcess.ipdl	Thu Oct 26 18:21:28 2023 +0000
+++ b/ipc/glue/PUtilityProcess.ipdl	Thu Oct 26 18:21:28 2023 +0000
@@ -12,6 +12,7 @@
 
 #if defined(XP_WIN)
 include protocol PWindowsUtils;
+include protocol PWinFileDialog;
 #endif
 
 #if defined(MOZ_SANDBOX) && defined(MOZ_DEBUG) && defined(ENABLE_TESTS)
@@ -102,6 +103,7 @@
 
 #if defined(XP_WIN)
   async StartWindowsUtilsService(Endpoint<PWindowsUtilsChild> aEndpoint);
+  async StartWinFileDialogService(Endpoint<PWinFileDialogChild> aEndpoint);
 
   async GetUntrustedModulesData() returns (UntrustedModulesData? data);
 